.github/workflows/preview-ready.yaml

```diff
@@ -1,4 +1,4 @@
-name: Integration Test Preview
+name: Preview Ready
```

Comment:
I tried to add performance tests to run here after integration tests, but no matter how low I set the load there are failures like this 
https://github.com/Basis-Theory-Labs/send-securely/runs/6600019649?check_suite_focus=true